src/test/java/uk/nhs/hee/tis/trainee/forms/service/LtftServiceTest.java

```diff
@@ -148,4 +162,145 @@ void shouldCountFilteredLtftWhenFiltersNotEmpty() {
     assertThat("Unexpected count.", count, is(40L));
     verify(ltftRepository, never()).count();
   }
+
+  @Test
+  void shouldReturnEmptyIfLtftFormNotFound() {
+    when(ltftRepository.findByTraineeTisIdAndId(any(), any())).thenReturn(Optional.empty());
+
+    Optional<LtftFormDto> formDtoOptional = service.getLtftForm(ID);
+
+    assertThat("Unexpected form returned.", formDtoOptional.isEmpty(), is(true));
+    verify(ltftRepository).findByTraineeTisIdAndId(any(), eq(ID));
+    verifyNoMoreInteractions(ltftRepository);
+  }
+
+  @Test
+  void shouldReturnEmptyIfLtftFormForTraineeNotFound() {
+    when(ltftRepository.findByTraineeTisIdAndId(TRAINEE_ID, ID)).thenReturn(Optional.empty());
+
+    Optional<LtftFormDto> formDtoOptional = service.getLtftForm(ID);
+
+    assertThat("Unexpected form returned.", formDtoOptional.isEmpty(), is(true));
+    verify(ltftRepository).findByTraineeTisIdAndId(TRAINEE_ID, ID);
+    verifyNoMoreInteractions(ltftRepository);
+  }
+
+  @Test
+  void shouldReturnDtoIfLtftFormForTraineeFound() {
+    LtftForm form = new LtftForm();
+    form.setId(ID);
+    form.setTraineeTisId(TRAINEE_ID);
+    form.setName("test");
+    when(ltftRepository.findByTraineeTisIdAndId(TRAINEE_ID, ID))
+        .thenReturn(Optional.of(form));
+
+    Optional<LtftFormDto> formDtoOptional = service.getLtftForm(ID);
+
+    assertThat("Unexpected form returned.", formDtoOptional.isEmpty(), is(false));
+    verify(ltftRepository).findByTraineeTisIdAndId(TRAINEE_ID, ID);
+    LtftFormDto returnedFormDto = formDtoOptional.get();
+    assertThat("Unexpected returned LTFT form.", returnedFormDto.equals(mapper.toDto(form)),
```

Comment:
As previous, not sure if there's a reason not to lean on the `is()` matcher instead of equals=true